File: Sep3-SemesterProject/Tier2-BusinessLogic/src/main/java/via/sdj3/proofofconcept_v3/Dto/RegisterPatientDto.java
package via.sdj3.proofofconcept_v3.Dto;

import com.fasterxml.jackson.annotation.JsonProperty;

public class RegisterPatientDto {
    @JsonProperty("UserName")
    private String UserName;

    @JsonProperty("Password")
    private String Password;

    @JsonProperty("FullName")
    private String FullName;

    public RegisterPatientDto(String fullName, String userName, String password) {
        this.FullName = fullName;
        this.UserName = userName;
        this.Password = password;
    }

    public RegisterPatientDto() {
    }

    public String getUserName() {
        return UserName;
    }

    public void setUserName(String userName) {
        this.UserName = userName;
    }

    public String getFullName() {
        return FullName;
    }

    public void setFullName(String fullName) {
        this.FullName = fullName;
    }

    public String getPassword() {
        return Password;
    }

    public void setPassword(String password) {
        this.Password = password;
    }
}

File: Sep3-SemesterProject/Tier1-Presentation/Domain/DTOs/RegisterPatientDto.cs
using Domain.Models;

namespace Domain.DTOs;

public class RegisterPatientDto
{
    public string UserName { get;set; }
    public string Password { get; set; }
    public string FullName { get; set; }
    public List<Appointment> Appointments { get; }

    public RegisterPatientDto(string name, string username, string password)
    {
        FullName = name;
        UserName = username;
        Password = password;
        Appointments = new List<Appointment>();
    }
}
